frontend/src/metabase/querying/notebook/components/DataStep/DataStep.unit.spec.tsx

```diff
@@ -389,7 +403,7 @@ describe("DataStep", () => {
       it("should not show the tooltip", async () => {
         setup({ isEmbeddingSdk: true });
 
-        await userEvent.hover(screen.getByText("Orders"));
+        await userEvent.hover(await screen.findByText("Orders"));
```

Comment:
the UI won't show up immediately anymore. We'll have to wait for an API response to see the number of available tables and orders, so we could determine if we want to show the simple data picker or the multi-stage data picker.